src/Maestro/Client/src/Generated/Models/UpdateFrequency.cs

```diff
@@ -13,5 +13,9 @@ public enum UpdateFrequency
         EveryDay,
         [EnumMember(Value = "everyBuild")]
         EveryBuild,
+        [EnumMember(Value = "twiceDaily")]
```

Comment:
I see. It compiled because I manually updated the Maestro code with the new enum values. If I run the client code gen, I get the compiler errors in darc that you were expecting.

![image](https://user-images.githubusercontent.com/2696087/57154622-33999e80-6d8e-11e9-9749-c45295ed1141.png)